tests/backup/backup_restore_basic_test.go

```diff
@@ -2763,3 +2764,446 @@ var _ = Describe("{BackupRestoreOnDifferentK8sVersions}", func() {
 	})
 })
+
+// BackupCRsThenMultipleRestoresOnHigherK8sVersion deploys CRs (CRD + webhook) -> backups them up -> creates two simulatanous restores on a cluster with higher K8s version :: one restore is Success and other PartialSuccess
+var _ = Describe("{BackupCRsThenMultipleRestoresOnHigherK8sVersion}", func() {
+
+	var (
+		backupNames          []string             // backups in px-backup
+		restoreNames         []string             // restores in px-backup
+		restoreLaterNames    []string             // restore-laters in px-backup
+		scheduledAppContexts []*scheduler.Context // Each Context is for one Namespace which corresponds to one App
+		preRuleNameList      []string
+		postRuleNameList     []string
+		scheduledClusterUid  string
+		cloudCredName        string
+		cloudCredUID         string
+		backupLocationUID    string
+		backupLocationName   string
+	)
+
+	var (
+		appList           = Inst().AppList
+		namespaceMapping  = make(map[string]string)
+		backupLocationMap = make(map[string]string)
+		labelSelectors    = make(map[string]string)
+		providers         = getProviders()
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupCRsThenMultipleRestoresOnHigherK8sVersion", "deploy CRs (CRD + webhook); then backup; create two simulatanous restores on cluster with higher K8s version; one restore is Success and other PartialSuccess", nil, 83716)
+	})
+
+	It("Deploy CRs (CRD + webhook); Backup; two simulatanous Restores with one Success and other PartialSuccess. (Backup and Restore on different K8s version)", func() {
+
+		defer func() {
+			log.InfoD("switching to default context")
+			err := SetClusterContext("")
+			log.FailOnError(err, "failed to SetClusterContext to default cluster")
+		}()
+
+		Step("Verify if app used to execute test is a valid/allowed spec (apps) for *this* test", func() {
+			log.InfoD("specs (apps) allowed in execution of test: %v", appsWithCRDsAndWebhooks)
+			for i := 0; i < len(appList); i++ {
+				contains := Contains(appsWithCRDsAndWebhooks, appList[i])
+				dash.VerifyFatal(contains, true, fmt.Sprintf("app [%s] allowed in execution of this test", appList[i]))
+			}
+		})
+
+		Step("creating source and destination cluster", func() {
+			log.InfoD("creating source and destination cluster")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, "create source and destination cluster")
+
+			clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+
+			scheduledClusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, SourceClusterName)
```

Comment:
I named it the same as the parameter of `ValidateBackup` to which this is gicen as an argument (in the future) so that it's clear as to what is being validated (deeper validation).
see https://github.com/portworx/torpedo/pull/1341#discussion_r1176139753

Reply:
It would be more clean if we name is source and destination. We would know where it is being scheduled by looking at the test case.

Reply:
Please change this

Reply:
I missed this. Will do now.